docs/getting-started.md

```diff
@@ -254,8 +272,10 @@ scanner on the `dependencies` scope in this tutorial. If you also want to scan t
 advised to configure a cache for the scan results as documented in the [README](../README.md) to speed up repeated
 scans.
 
+Because during _analyzer_ step we provided `.ort.yml` config file to exclude `devDependencies`, we just need to provide `--skip-excluded` parameter to scan only `dependencides` scope
```

Comment:
Nit (passive wording): "As duing the _analyzer_ step an `.ort.yml` configuration file was provided to exclude `devDependencies`, the `--skip-excluded` can be used to avoid the download and scanning of that scope."